crates/bin/pcli/Cargo.toml

```diff
@@ -106,7 +106,7 @@ tempfile = "3.3.0"
 regex = "1.6.0"
 penumbra-governance            = { path = "../../core/component/governance", default-features = false }
 penumbra-proof-params = { path = "../../crypto/proof-params", features = [
-    "proving-keys",
+    "bundled-proving-keys",
     "download-proving-keys",
```

Comment:
should `download-proving-keys` imply `bundled-proving-keys`? probably, right? and in that case maybe it should be `download-bundled-proving-keys`?

Reply:
Ah, never mind, I missed that this was in the dev-dependencies section.